src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureFilter.h

```diff
@@ -36,6 +36,7 @@
 
 #include <OpenMS/ANALYSIS/OPENSWATH/MRMFeatureQC.h>
 #include <OpenMS/ANALYSIS/TARGETED/TargetedExperiment.h>
+#include <OpenMS/ANALYSIS/QUANTITATION/AbsoluteQuantitationMethod.h>
```

Comment:
you should be able to forward-declare `AbsoluteQuantitationMethod` and avoid the include

Reply:
I am not sure I see how forward-declaration would be desirable here nor the benefit of avoiding the include.  Can you elaborate?

Reply:
its all about compile time.
Unrelated code will not get pulled in compile units which include MRMFeatureFilter.h and speed up compilation.